relay-profiling/src/sample/v2.rs

```diff
@@ -165,4 +176,35 @@ mod tests {
         let second_parse = parse(&second_payload[..]);
         assert!(second_parse.is_ok(), "{:#?}", second_parse);
     }
+
+    #[test]
+    fn test_samples_sorted() {
+        let mut chunk = ProfileData {
+            samples: vec![
+                Sample {
+                    stack_id: 0,
+                    thread_id: "1".into(),
+                    timestamp: FiniteF64::new(2000.0).unwrap(),
+                },
+                Sample {
+                    stack_id: 0,
+                    thread_id: "1".to_string(),
+                    timestamp: FiniteF64::new(1000.0).unwrap(),
+                },
+            ],
+            ..Default::default()
+        };
+
+        chunk.sort_samples_by_timestamp();
```

Comment:
This should call `normalize`, the public interface not the implementation detail (e.g. the test doesn't make sure normalize calls into the sort function)